### PR TITLE
Fix OCI repository URL resolution

### PR DESCRIPTION
**Pull request by knqyf263**
Same as https://github.com/openvex/discovery/pull/40

### CODE REVIEW

**Review by puerco — COMMENTED** [review]
Thanks Teppei! Just a small nit but otherwise lgtm :point_down:

File: pkg/ctl/implementation.go [hunk 1]
```diff
@@ -551,7 +551,7 @@ func (impl *defaultVexCtlImplementation) NormalizeProducts(subjects []productRef
 			ref := ""
 			qs := p.Qualifiers.Map()
 			if r, ok := qs["repository_url"]; ok {
-				ref = fmt.Sprintf("%s/%s", strings.TrimSuffix(r, "/"), p.Name)
+				ref = r
```
[the remark this answers is not in the document]
Comment (knqyf263) [reply pkg/ctl/implementation.go]: Yes, I was wondering about that. Also, I added a test.
https://github.com/openvex/vexctl/pull/235/commits/32d37a1d4be3293938ddc8fd3d5a98c195d48af6

**Review by puerco — APPROVED** [review]
Oooops sorry @knqyf263 i went on vacation and forgot to finish this review. 

Thank you for fixing this. LGTM!

**Comment by knqyf263** [comment]
No problem. I've also been on vacation. Thanks!

**Pull request merged**